### DISCUSSION TITLE
[resolved] Tesla wakes up everyday at the same time

### DISCUSSION THREAD

**Discussion by victorcerutti** [teslamate-org/teslamate#2105] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

First of all thank you for Teslamate. This is really a fantastic project

I own a Tesla (model 3 LR 2019) for 10 days now and set up Teslamate a few days ago.
I read every issues and posts about failing to fall asleep and I think my case is a bit different

My battery loses about 4% per night without any particular reasons (no sentry, etc…)
It seems that it make time to fall asleep and there is a clear pattern
<img width="1370" alt="Capture d’écran 2021-10-26 à 12 04 37" src="https://user-images.githubusercontent.com/3236171/139036441-1af1c795-69c3-4077-b175-4e57a6658565.png">
<img width="1364" alt="Capture d’écran 2021-10-26 à 12 04 47" src="https://user-images.githubusercontent.com/3236171/139036449-fef619ff-2467-41e2-afba-d3f0aef41be7.png">
<img width="1369" alt="Capture d’écran 2021-10-27 à 09 41 18" src="https://user-images.githubusercontent.com/3236171/139036495-42d2c41a-d3e0-4364-8498-f48aca859c1f.png">

Every day, it fails to go asleep until after midnight and goes awake just before 6 in the morning
See the log attached

Is there any reason why Teslamate seems to connect to the car and awakes it at this particular time of the day ? Is the docker container restarting ?
Is it related to the fact it doesn't go to sleep before midnight ?

I will continue my investigations and report my findings

### Expected Behavior

Car not awaken at 6am

### Relevant log output

```shell
Attaching to teslamate_teslamate_1
teslamate_1  | 2021-10-27 05:59:22.427 car_id=1 [info] Start / :online
teslamate_1  | 2021-10-27 05:59:22.452 car_id=1 [info] Connecting ...
teslamate_1  | 2021-10-27 06:02:33.624 car_id=1 [info] Suspending logging
teslamate_1  | 2021-10-27 06:32:34.008 car_id=1 [info] Vehicle is still online. Falling asleep for: 33 min
teslamate_1  | 2021-10-27 07:02:35.303 car_id=1 [info] Vehicle is still online. Falling asleep for: 1 h 3 min
teslamate_1  | 2021-10-27 07:32:38.331 car_id=1 [info] Vehicle is still online. Falling asleep for: 1 h 33 min`
```

### Type of installation

Docker

### Version

1.24.1

**Comment by adriankumpf** [comment, marked as answer]
TeslaMate does not send any commands, and is not able to wake up your car. There have been other threads on this topic already, so I moved the issue to a discussion.

**Reply by victorcerutti** [reply]
I get it but do you see any reasons why the wake-up time (back online a 6am every morning) seems to be related to the container being restarted ?

**Reply by victorcerutti** [reply]
@adriankumpf 
So... I tried a lot of thing since the last few days
(Everyday the car would sleep from midnight to 6am and no sleep at all during the rest of the day)
I finally found out that it was the option "summon" from FSD which was active. It seems that it keeps the car online except for a specific time at night.
After disabling the feature the car now goes to sleep a few minutes after driving

I am sorry I thought Teslamate was the culprit. I only have the car for 2 weeks now and still need to get used to it :)

Thank you for the support and for the continuous development of Teslamate
